### PR TITLE
Update for task with past due date to be removed from dashboard

### PR DESCRIPTION
**Pull request by shivamssin**
Updated changes for removal of task with past due date.

### CODE REVIEW

File: DCode.Data/RequestorRepository/RequestorRepository.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public IEnumerable<task> GetTaskApplicantsForApproval(int selectedTaskTypeId, in
             //Pick records based on the pageindex.
             var filteredRecords = tasks.OrderByDescending(x => x.CREATED_ON).Skip((currentPageIndex - 1) * recordsCount).Take(recordsCount);
             var end = DateTime.Now - start;
-            return filteredRecords.ToList();
+            return filteredRecords.ToList().Where(x => x.DUE_DATE >= DateTime.Now);
```
Comment (ritwiksen) [inline DCode.Data/RequestorRepository/RequestorRepository.cs]: Please execute Where before the ToList().

File: DCode.Services/Contributor/ContributorService.cs [hunk 2]
```diff
@@ -230,7 +231,14 @@ public TaskResponse GetAllTasks(string searchKey, int currentPageIndex, int reco
                 tasks = _taskSkillModelFactory.CreateModelList<Models.ResponseModels.Task.Task>(dbTaskSkills);
             }
 
-            taskList.Tasks = tasks;
+            foreach (var task in tasks)
```
Comment (ritwiksen) [inline DCode.Services/Contributor/ContributorService.cs]: Please ensure querying is done in the Repo layer.

**Review by ritwiksen — CHANGES_REQUESTED** [review]

**Review by ritwiksen — APPROVED** [review]
Looks good

**Review by shivam-r — CHANGES_REQUESTED** [review]
Need to check the status while doing the filter based on due date.

Requestor and contributor should be able to see the tasks past due date in their history tab if task is assigned or approved.

**Comment by ritwiksen** [comment]
This PR is no longer relevant and can be closed.

**Pull request closed without merging**